.github/workflows/reusable-tsan.yml

```diff
@@ -57,7 +55,12 @@ jobs:
         save: ${{ github.event_name == 'push' }}
         max-size: "200M"
     - name: Configure CPython
-      run: ${{ inputs.options }}
+      run: >-
+        ./configure
+        --config-cache
+        --with-thread-sanitizer
+        --with-pydebug
+        ${{ fromJSON(inputs.free-threading) && '--disable-gil' || '' }}
```

Comment:
:thinking: is it significant that I moved the optional argument towards the end? I would imagine the order shouldn't matter. Is the failure unrelated to this patch..?

Reply:
The failure happens when re-running (so isn't temporary) and isn't happening on `main`.